<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EDIT_MAintenance</class>
 <widget class="QDialog" name="EDIT_MAintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Esprit-RP : Product management</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>621</x>
     <y>495</y>
     <width>80</width>
     <height>58</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Ressources.qrc">
     <normaloff>:/Icones/Ressources/save-23.png</normaloff>:/Icones/Ressources/save-23.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>495</y>
     <width>80</width>
     <height>58</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Ressources.qrc">
     <normaloff>:/Icones/Ressources/exit.png</normaloff>:/Icones/Ressources/exit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>73</width>
     <height>54</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>NAME</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>TO_DO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_ID"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Name"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Status"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_todo"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_type"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
